values.go

```diff
@@ -1617,12 +1636,18 @@ func (v Dictionary) WithType(dictionaryType *DictionaryType) Dictionary {
 }
 
 func (v Dictionary) ToGoValue() any {
-	ret := map[any]any{}
+	ret := map[string]any{}
```

Comment:
Why change the key type to always being a string? What if the key type of the dictionary is e.g. `Int`?

Reply:
Another thing left over from overflow. Will fix.